### DISCUSSION TITLE
about oracle

### DISCUSSION THREAD

**Discussion by yangmeng66** [dbeaver/dbeaver#16903] [Q&A]
Ask whether Oracle will support debugging in the future?

**Comment by LonwoLonwo** [comment]
We have ticket for this: https://github.com/dbeaver/dbeaver/issues/3404

You can vote for the topic.

**Comment by yangmeng66** [comment]
Thank you very much. I hope Oracle can have the debugging function as soon as possible. I am now trying to develop the debugging of our byte database plug-in (similar to Postgres)
